### PR TITLE
Fix erblint errors on the `app/views/liquids` folder

### PR DESCRIPTION
**Pull request by AlbertoPdRF**
## What type of PR is this? (check all applicable)

- [ ] Refactor
- [ ] Feature
- [ ] Bug Fix
- [x] Optimization
- [ ] Documentation Update

## Description

This PR fixes the errors resulting from running `bundle exec erblint app/views/liquids`.

## Related Tickets & Documents

This PR is part of #8370.

## Added tests?

- [ ] yes
- [x] no, because they aren't needed
- [ ] no, because I need help

## Added to documentation?

- [ ] docs.dev.to
- [ ] readme
- [x] no documentation needed

## [optional] What gif best describes this PR or how it makes you feel?

Cleaning gif

### CODE REVIEW

File: app/views/liquids/_stackexchange.html.erb [hunk 1]
```diff
@@ -6,11 +6,11 @@
         <h1>
           <img src="<%= ActionController::Base.helpers.asset_path(logo_to_use) %>" alt="">
             <a href="<%= post_url %>" target="_blank" rel="noopener noreferrer">
-              <%= post_type == "question" ? "" : "<span class='title-flare'>answer</span> re: ".html_safe %> <%= CGI::unescapeHTML title %>
+              <%= post_type == "question" ? "" : "<span class='title-flare'>answer</span> re: ".html_safe %> <%= CGI.unescapeHTML title %>
             </a>
         </h1>
         <div class="ltag__stackexchange--post-metadata">
-          <span><%= Time.at(created_at).strftime("%b %e '%y") %></span>
+          <span><%= Time.zone.at(created_at).strftime("%b %e '%y") %></span>
```
Comment (rhymes) [inline app/views/liquids/_stackexchange.html.erb:13]: 🔥 yes!

**Comment by AlbertoPdRF** [comment]
The second commit makes tests pass (at least locally) but I don't really get what's the problem 😕 Any explanation would be appreciated!

**Comment by rhymes** [comment]
@AlbertoPdRF unfortunately it's a known flaky spec. I restarted the build. Crossing fingers :D

**Comment by AlbertoPdRF** [comment]
The second failed build was due to a flaky spec, but was [the first one](https://travis-ci.com/github/thepracticaldev/dev.to/builds/170848230) too? That's the one I was referring to in my comment.

**Comment by rhymes** [comment]
ooh got it! yeah, that needs to be manually fixed.

Run the failing test locally, then run `approvals verify`, accept the change, re-run the test and push.

Let me know if you need further help

**Comment by AlbertoPdRF** [comment]
Thanks for the help @rhymes! Let me know if it looks good now 🙂

**Comment by rhymes** [comment]
@AlbertoPdRF could you kindly rebase on top of master? We likely fixed the problem with the parallel builds

**Comment by AlbertoPdRF** [comment]
Done!

**Pull request merged**